assets/js/container/register-maker/register-maker.js

```diff
@@ -285,11 +297,28 @@ class RegisterMaker extends React.Component {
     if (this.context.user && this.context.user.id) {
       // todo redirect to home?
     }
+
+    this.getCountryList("de");
   }
 
-  onSubmit = (data) => {
-    console.log(data);
+  getCountryList(lang) {
+    const url = `https://restcountries.eu/rest/v2/all?fields=alpha2Code;name;translations`;
```

Comment:
I think we should use this content in the repository and include it in the file instead of fetching it everytime. So we are independent from this service. If country codes change we need to update it manually.. but this won't happen too often I hope ;)

Reply:
I created #241 for this. Also country codes adhere to the  ISO standard so they won't change. But you are right we should fetch it from our own back-end.